src/navs/index.js

```diff
@@ -3988,66 +3988,164 @@ export const docsMenu = {
+                    name: 'Privacy mode',
+                    url: '/docs/llm-analytics/privacy-mode',
+                    icon: 'IconLock',
+                    color: 'purple',
+                    featured: true,
                 },
                 {
-                    name: 'Traces and generations',
-                    url: '/docs/ai-engineering/traces-generations',
-                    icon: 'IconTarget',
+                    name: 'Concepts',
+                },
+                {
+                    name: 'Generations',
+                    url: '/docs/llm-analytics/generations',
+                    icon: 'IconAIText',
+                    color: 'seagreen',
+                    featured: true,
+                },
+                {
+                    name: 'Traces',
+                    url: '/docs/llm-analytics/traces',
+                    icon: 'IconUserPaths',
                     color: 'orange',
+                    featured: true,
+                },
+                {
+                    name: 'Spans',
+                    url: '/docs/llm-analytics/spans',
+                    icon: 'IconBrackets',
+                    color: 'blue',
+                    featured: true,
+                },
+                {
+                    name: 'Embeddings',
+                    url: '/docs/llm-analytics/embeddings',
+                    icon: 'IconDatabase',
+                    color: 'purple',
+                    featured: true,
+                },
+                {
+                    name: 'Calculating LLM costs',
+                    url: '/docs/llm-analytics/calculating-costs',
+                    icon: 'IconBalance',
+                    color: 'green',
+                    featured: true,
+                },
+                {
+                    name: 'Guides',
+                },
+                {
+                    name: 'Analyze LLM performance',
+                    url: '/docs/llm-analytics/dashboard',
+                    icon: 'IconDashboard',
+                    color: 'red',
                 },
                 {
                     name: 'Tutorials and guides',
-                    url: '/docs/ai-engineering/tutorials',
+                    url: '/docs/llm-analytics/tutorials',
                     icon: 'IconGraduationCap',
                     color: 'salmon',
                     featured: true,
                 },
                 {
-                    name: 'Observability integrations',
-                    url: '/docs/ai-engineering/llm-insights',
-                    icon: 'IconAIText',
-                    featured: true,
+                    name: 'Resources',
+                },
+                {
+                    name: 'Troubleshooting and FAQs',
+                    url: '/docs/llm-analytics/troubleshooting',
+                    icon: 'IconQuestion',
                     color: 'purple',
+                },
+                {
+                    name: 'Integrations',
```

Comment:
I don't think I like these integrations in resources. Resources are things that provide supporting information and details. These seem like entire use cases.

But I trust y'alls judgement